account_multicurrency_revaluation/report/currency_unrealized_report.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2012-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, models
+
+
+class ShellAccount(object):
+
+    # Small class that avoid to override account account object
```

Comment:
move this to docstring following [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)